README.md

```diff
@@ -48,10 +48,10 @@ _Last updated by forest2001 on 2025-04-01_
 
 1. Must wear ESU uniform and use ESU vehicles only.
 2. Cannot be armed or commit crimes.
-3. Immune to crimes being committed against them.
+3. Protected while clearly performing ESU duties; this is not a license to commit crimes or avoid lawful enforcement.
```

Comment:
Clarify what ESU are protected from. Also change "performing ESU duties" wording to make sure ESU aren't fair game if they're just patrolling.